### PR TITLE
Stat uniform tests

### PR DESCRIPTION
**Pull request by kczimm**
Created test file for Uniform random variables. The test file was based off of the ChiSquared test file.

### CODE REVIEW

File: stat/distuv/uniform_test.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// Copyright ©2016 The Gonum Authors. All rights reserved.
```
Comment (btracey) [inline stat/distuv/uniform_test.go]: 2017

File: stat/distuv/uniform_test.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+		min, max, x, want float64
+	}{
+		{0, 1, 1, 1},
+		{2, 4, 0, 0},
+		{2, 4, 5, 0},
+		{2, 4, 3, 0.5},
+		{0, 100, 1, 0.01},
+	} {
+		pdf := Uniform{test.min, test.max, nil}.Prob(test.x)
+		if !floats.EqualWithinAbsOrRel(pdf, test.want, 1e-10, 1e-10) {
```
Comment (btracey) [inline stat/distuv/uniform_test.go]: These tolerances should be higher, there basically shouldn't be error. 1e-15 instead

File: stat/distuv/uniform_test.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		min, max, x, want float64
+	}{
+		{0, 1, 1, 1},
+		{0, 100, 100, 1},
+		{0, 100, 0, 0},
+		{0, 100, 50, 0.5},
+		{0, 50, 10, 0.2},
+	} {
+		cdf := Uniform{test.min, test.max, nil}.CDF(test.x)
+		if !floats.EqualWithinAbsOrRel(cdf, test.want, 1e-10, 1e-10) {
```
Comment (btracey) [inline stat/distuv/uniform_test.go]: Same here.

File: stat/distuv/uniform_test.go [hunk 4]
```diff
@@ -0,0 +1,74 @@
+// Copyright ©2016 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math/rand"
```
Comment (btracey) [inline stat/distuv/uniform_test.go]: Could you make this "golang.org/x/exp/rand" please?

File: stat/distuv/uniform_test.go (hunk 1 above)
Comment (kczimm) [reply stat/distuv/uniform_test.go]: done

File: stat/distuv/uniform_test.go (hunk 2 above)
Comment (kczimm) [reply stat/distuv/uniform_test.go]: done

File: stat/distuv/uniform_test.go (hunk 3 above)
Comment (kczimm) [reply stat/distuv/uniform_test.go]: done

File: stat/distuv/uniform_test.go (hunk 4 above)
Comment (kczimm) [reply stat/distuv/uniform_test.go]: Is this a direct replacement for math/rand? Does this mean that `uniform.go` needs to be changed, as well?

**Comment by btracey** [comment]
We have https://github.com/gonum/gonum/pull/301 to fix the rest. Let's get that in first, and then merge this one. Sorry for the delay.

**Comment by kczimm** [comment]
Sounds good. I didn't see #301.

**Comment by btracey** [comment]
Hi @kczimm sorry for the delay on this one. Could you bring this up to date with master and make another PR?

**Comment by kczimm** [comment]
How's that?

**Comment by btracey** [comment]
Thanks!

**Pull request merged**